### PR TITLE
OJ-1825: Added unhappy path tests

### PR DESCRIPTION
**Pull request by surajkumar**
## Proposed changes
Added unhappy path scenario tests to nino-check.

### What changed

Created a new test file to house the failure scenarios. 

### Why did it change
Testing

### Issue tracking

- [OJ-1825](https://govukverify.atlassian.net/browse/OJ-1825)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[OJ-1825]: https://govukverify.atlassian.net/browse/OJ-1825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: integration-tests/tests/aws/nino-check-unhappy.test.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          lastName: testUser.lastName,
+        });
+        const startExecutionResult = await executeStepFunction({
+          sessionId: sessionId,
+          nino: "bad-good-nino",
+        });
+        await clearItems(process.env.NINO_USERS_TABLE as string, {
+          nino: goodBadNino,
+        });
+        expect(startExecutionResult.output).toBe(undefined);
```
Comment (jkunle) [inline integration-tests/tests/aws/nino-check-unhappy.test.ts:90]: `toBeUndefined`

File: integration-tests/tests/aws/nino-check-unhappy.test.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      id: sessionId,
+    });
+  });
+
+  describe("Unhappy User Input Paths", () => {
+    it("should fail when there is no sessionId present", async () => {
+      const startExecutionResult = await executeStepFunction({
+        nino: nino,
+      });
+      expect(startExecutionResult.output).toBe('{"sessionId":"12345"}');
```
Comment (jkunle) [inline integration-tests/tests/aws/nino-check-unhappy.test.ts:36]: is this correct?

File: integration-tests/tests/aws/nino-check-unhappy.test.ts (hunk 1 above)
Comment (surajkumar) [reply integration-tests/tests/aws/nino-check-unhappy.test.ts]: Updated

File: integration-tests/tests/aws/nino-check-unhappy.test.ts (hunk 2 above)
Comment (jkunle) [reply integration-tests/tests/aws/nino-check-unhappy.test.ts:36]: Ok, so either this is wrong or the step function local version is wrong. I think the state machine needs to be checked to be sure

Comment (surajkumar) [reply integration-tests/tests/aws/nino-check-unhappy.test.ts]: I see what you mean now, my typo!

**Review by jkunle — APPROVED** [review]

**Pull request merged**